src/documents/activities/Create-a-Thon.md

```diff
@@ -6,49 +6,73 @@ eleventyNavigation:
+A _CaT_ encourages participants to collaboratively build an artifact by “designing by metaphor” and use it as a means to
+translate their ideas to the systems, products or services being designed or tested. This helps constrain the physical
+artefact and enables participants to draw out connections that wouldn't otherwise be clear if they were directly asked
+to design a solution. It also gives non-expert designers an easier way to get involved and contribute their ideas
+without having prior knowledge or expertise.
 
-During this collaborative event, participants should continuously describe the process to each other, support each other when there are different abilities, and communicate how the work gets done. Constant negotiation of needs generates individual stories and valuable information that should be recorded, collected, and applied to enhance the design of the system, product or service being iterated upon. It is important to avoid distilling user insights and stories to a collection of averages and eliminating individual differences in favour of sameness, because each story is as important as another and will contribute to rich designs and insights.
+During this collaborative event, participants should continuously describe the process to each other, support each other
+when there are different abilities, and communicate how the work gets done. Constant negotiation of needs generates
+individual stories and valuable information that should be recorded, collected, and applied to enhance the design of the
+system, product or service being iterated upon. It is important to avoid distilling user insights and stories to a
+collection of averages and eliminating individual differences in favour of sameness, because each story is as important
+as another and will contribute to rich designs and insights.
 
-It is possible, even desirable, to conduct several _CaTs_ at different stages throughout a design process to gain additional insights.
+It is possible, even desirable, to conduct several _CaTs_ at different stages throughout a design process to gain
+additional insights.
 
 <figure>
-[![An image of a Create-a-Thon participant describing their group's design.](/assets/images/CaT-02.jpg)](/assets/images/CaT-02.jpg)
+[![An image of a Create-a-Thon participant describing their group's
+design.](/assets/images/CaT-02.jpg)](/assets/images/CaT-02.jpg)
 <figcaption>
-*Figure 2:* A participant shares their group's design after being given the task of creating a software version of the physical marble track toy.</figcaption>
+*Figure 2:* A participant shares their group's design after being given the task of creating a software version of the
+physical marble track toy.</figcaption>
```

Comment:
The second line "physical marble track toy." is showing up on a separate line on the rendered page.
![Screen Shot 2020-12-07 at 11 44 45 AM](https://user-images.githubusercontent.com/556024/101378880-9e239200-3881-11eb-9bc4-36aa8c6980b2.png)